File: PHP/Program/DPR.php
<?php

class AnggotaDPR {
    private $id;
    private $nama;
    private $bidang;
    private $partai;
    private $fotoProfil;

    public function getId() {
        return $this->id;
    }

    public function setId($id) {
        $this->id = $id;
    }

    public function getNama() {
        return $this->nama;
    }

    public function setNama($nama) {
        $this->nama = $nama;
    }

    public function getBidang() {
        return $this->bidang;
    }

    public function setBidang($bidang) {
        $this->bidang = $bidang;
    }

    public function getPartai() {
        return $this->partai;
    }

    public function setPartai($partai) {
        $this->partai = $partai;
    }

    public function getFotoProfil() {
        return $this->fotoProfil;
    }

    public function setFotoProfil($fotoProfil) {
        $this->fotoProfil = $fotoProfil;
    }

    public function __construct($id, $nama, $bidang, $partai, $fotoProfil) {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->fotoProfil = $fotoProfil;
    }

    public static function headerInfo() {
        // Menampilkan tabel informasi anggota DPR
        
        echo "<tr>";
        echo "<th>ID</th>";
        echo "<th>Nama</th>";
        echo "<th>Bidang</th>";
        echo "<th>Partai</th>";
        echo "<th>Foto Profil</th>";
        echo "</tr>";
    }

    public function tampilkanInfo() {
        echo "<tr>";
        echo "<td>" . $this->getId() . "</td>";
        echo "<td>" . $this->getNama() . "</td>";
        echo "<td>" . $this->getBidang() . "</td>";
        echo "<td>" . $this->getPartai() . "</td>";
        echo "<td><img src='" . $this->getFotoProfil() . "' alt='Foto Profil' width='90' height='120'></td>";
        echo "</tr>";
    }
}

?>

File: Java/Program/DPR.java
//Class delaration. For Java, the first letter MUST BE capitalized.
public class DPR
{
    //Private attributws.
    private String id; // Atribut unruk menampung id anggota DPR
    private String nama; // Atribut unruk menampung nama anggota DPR
    private String bidang; // Atribut unruk menampung bidang anggota DPR
    private String partai; // Atribut unruk menampung partai anggota DPR

    /*Constructors */

    //Constructors. Take note that it doesn't have any return type
    public DPR()
    {
        this.id = "";
        this.nama = "";
        this.bidang = "";
        this.partai = "";
    }

    //Another constructore with parameter.
    public DPR(String id, String nama, String bidang, String partai)
    {
        this.id = id;
        this.nama = nama;
        this.bidang = bidang;
        this.partai = partai;
    }

    /* Getter and Setter */
    
    //Get id.
    public String getId(){
        return this.id;
    }

    //Set id.
    public void setId(String id)
    {
        this.id = id;
    }

    //Get nama.
    public String getnama(){
        return this.nama;
    }

    //Set nama.
    public void setnama(String nama)
    {
        this.nama = nama;
    }

    //Get bidang.
    public String getbidang(){
        return this.bidang;
    }

    //Set bidang.
    public void setbidang(String bidang)
    {
        this.bidang = bidang;
    }

    //Get partai.
    public String getpartai(){
        return this.partai;
    }

    //Set partai.
    public void setpartai(String partai)
    {
        this.partai = partai;
    }

    // Method to display member information
    public void displayInfo() {
        System.out.println("ID: " + id);
        System.out.println("Nama: " + nama);
        System.out.println("Bidang: " + bidang);
        System.out.println("Partai: " + partai);
    }

    
}